### PR TITLE
Backend open ai endpoint

### LINKED ISSUE #16: Open Ai romanization of lyrics
Need to create endpoint for chatgpt, using 4o mini and be able to send request and also batched requests of language and their language codes, in LRC format to have it translate back into LRC format but with romanization. Need to do testing with prompts and make sure it doesn't exceed maximum token usage on batch calls?

### PR DESCRIPTION
**Pull request by darint916**
# 🚀 Pull Request

## 📋 Summary

The finished endpoint allows for the request of LRC lyrics in native text and translates and romanizes them while keeping them in LRC format. Allows for toggling romanization and translation either or to speed up the model. Benchmarks are kept in there, and tests are ran in the screenshots below. Uses GPT 4o model.

## 🔍 Related Issues

Fixes #16 

## 📷 Screenshots (if applicable)

Swagger docs test
Input:
```
{
  "originalLyrics": "[00:00.15] La, la, la-la-la-la\n[00:07.88] 瞳の奥 重なる eyes\n[00:11.85] 今宵 二人を照らす moon\n[00:16.03] Nine to five dreaming, これじゃまるで恋ね\n[00:20.12] But the odds were high, 夢見たい気分?\n[00:25.29] 待ってるだけじゃ何も始まんない\n[00:29.37] I am precious, ooh, but, 自分の声で step up, boom\n[00:31.85] 時計の針が二人まで tick-tock\n[00:36.16] 魔法のように love in blooming (la-la-la-la)\n[00:41.36] 気づけば 抜け出せないよ, I'm good to go\n[00:46.04] (Go) yeah (yeah), 止めどなく夢中\n[00:49.00] You're my crush, 弾ける bubble\n[00:53.67] We can't stop, 惹かれ合うほど (oh)\n[00:58.21] Crush, 高鳴る鼓動\n[01:01.78] We can't stop, 君となら fallin' in love\n[01:05.43] Whoa-whoa, oh-oh, woah-whoa, oh-oh\n[01:09.49] 夢の中踊りたい sugar spot\n[01:13.51] Whoa-whoa, oh-oh, oh-woah-whoa\n[01:16.75] 壊れるほど crush on you, 君となら fallin' in\n[01:22.32] 焦る気持ち答えはまだ, baby? Love is danger\n[01:26.21] 慎重になるほどに every piece of me, my pleasure\n[01:30.46] Honey (honey) like roses (roses) 舞うように誘う\n[01:33.96] 揺れる想い like a butterfly (butterfly), 溶けてく moonlight\n[01:38.76] 待ってるなんてもう言わせないで\n[01:42.82] Hey, you're my precious one, 自分の声で step up, boom\n[01:45.88] 時計の針が二人まで tick-tock\n[01:50.03] 魔法のように love in blooming (la-la-la-la)\n[01:55.20] 気づけば 抜け出せないよ, I'm good to go\n[01:59.74] (Go) yeah (yeah), 止めどなく夢中\n[02:02.93] You're my crush, 弾ける bubble\n[02:07.58] We can't stop, 惹かれ合うほど (oh)\n[02:12.10] Crush, 高鳴る鼓動\n[02:15.60] We can't stop, 君となら fallin' in love\n[02:19.41] Whoa-whoa, oh-oh, woah-whoa, oh-oh\n[02:23.33] 夢の中踊りたい sugar spot\n[02:27.43] Whoa-whoa, oh-oh, oh-woah-whoa\n[02:30.70] 壊れるほど crush on you, 君となら fallin' in\n[02:36.69] Oh, hey, DIVE, これからどこ行く?\n[02:40.76] I think I'm finally done with you\n[02:44.57] 壊れるほど love crush\n[02:47.54] 止めどなく夢中\n[02:51.01] You're my crush, 弾ける bubble\n[02:56.78] We can't stop, 惹かれ合うほど (oh)\n[03:01.31] Crush, 高鳴る鼓動\n[03:04.82] We can't stop, 君となら fallin' in love\n[03:09.34] Crush on you, もう待てない blue moon (ah, yeah)\n[03:13.25] I'm finally feeling like I got my freedom\n[03:17.81] Crush on you, 君も夢中, yeah, yeah\n[03:21.86] Love crush, you're my crush\n[03:27.29] ",
  "languageCode": "ja",
  "romanize": true,
  "translate": false
}
```
Output:
```
{
  "originalLyrics": "[00:00.15] La, la, la-la-la-la\n[00:07.88] 瞳の奥 重なる eyes\n[00:11.85] 今宵 二人を照らす moon\n[00:16.03] Nine to five dreaming, これじゃまるで恋ね\n[00:20.12] But the odds were high, 夢見たい気分?\n[00:25.29] 待ってるだけじゃ何も始まんない\n[00:29.37] I am precious, ooh, but, 自分の声で step up, boom\n[00:31.85] 時計の針が二人まで tick-tock\n[00:36.16] 魔法のように love in blooming (la-la-la-la)\n[00:41.36] 気づけば 抜け出せないよ, I'm good to go\n[00:46.04] (Go) yeah (yeah), 止めどなく夢中\n[00:49.00] You're my crush, 弾ける bubble\n[00:53.67] We can't stop, 惹かれ合うほど (oh)\n[00:58.21] Crush, 高鳴る鼓動\n[01:01.78] We can't stop, 君となら fallin' in love\n[01:05.43] Whoa-whoa, oh-oh, woah-whoa, oh-oh\n[01:09.49] 夢の中踊りたい sugar spot\n[01:13.51] Whoa-whoa, oh-oh, oh-woah-whoa\n[01:16.75] 壊れるほど crush on you, 君となら fallin' in\n[01:22.32] 焦る気持ち答えはまだ, baby? Love is danger\n[01:26.21] 慎重になるほどに every piece of me, my pleasure\n[01:30.46] Honey (honey) like roses (roses) 舞うように誘う\n[01:33.96] 揺れる想い like a butterfly (butterfly), 溶けてく moonlight\n[01:38.76] 待ってるなんてもう言わせないで\n[01:42.82] Hey, you're my precious one, 自分の声で step up, boom\n[01:45.88] 時計の針が二人まで tick-tock\n[01:50.03] 魔法のように love in blooming (la-la-la-la)\n[01:55.20] 気づけば 抜け出せないよ, I'm good to go\n[01:59.74] (Go) yeah (yeah), 止めどなく夢中\n[02:02.93] You're my crush, 弾ける bubble\n[02:07.58] We can't stop, 惹かれ合うほど (oh)\n[02:12.10] Crush, 高鳴る鼓動\n[02:15.60] We can't stop, 君となら fallin' in love\n[02:19.41] Whoa-whoa, oh-oh, woah-whoa, oh-oh\n[02:23.33] 夢の中踊りたい sugar spot\n[02:27.43] Whoa-whoa, oh-oh, oh-woah-whoa\n[02:30.70] 壊れるほど crush on you, 君となら fallin' in\n[02:36.69] Oh, hey, DIVE, これからどこ行く?\n[02:40.76] I think I'm finally done with you\n[02:44.57] 壊れるほど love crush\n[02:47.54] 止めどなく夢中\n[02:51.01] You're my crush, 弾ける bubble\n[02:56.78] We can't stop, 惹かれ合うほど (oh)\n[03:01.31] Crush, 高鳴る鼓動\n[03:04.82] We can't stop, 君となら fallin' in love\n[03:09.34] Crush on you, もう待てない blue moon (ah, yeah)\n[03:13.25] I'm finally feeling like I got my freedom\n[03:17.81] Crush on you, 君も夢中, yeah, yeah\n[03:21.86] Love crush, you're my crush\n[03:27.29] ",
  "languageCode": "JA",
  "romanizedLyrics": "La, la, la-la-la-la  \n[00:07.88] Hitomi no oku kasanaru eyes  \n[00:11.85] Koyoi futari o terasu moon  \n[00:16.03] Nine to five dreaming, kore ja maru de koi ne  \n[00:20.12] But the odds were high, yume mitai kibun?  \n[00:25.29] Matteru dake ja nanimo hajimannai  \n[00:29.37] I am precious, ooh, but, jibun no koe de step up, boom  \n[00:31.85] Tokei no hari ga futari made tick-tock  \n[00:36.16] Mahō no yō ni love in blooming (la-la-la-la)  \n[00:41.36] Kizukeba nukedasenai yo, I'm good to go  \n[00:46.04] (Go) yeah (yeah), tomedonaku muchū  \n[00:49.00] You're my crush, hajikeru bubble  \n[00:53.67] We can't stop, hikareau hodo (oh)  \n[00:58.21] Crush, takanaru kodō  \n[01:01.78] We can't stop, kimi to nara fallin' in love  \n[01:05.43] Whoa-whoa, oh-oh, woah-whoa, oh-oh  \n[01:09.49] Yume no naka odoritai sugar spot  \n[01:13.51] Whoa-whoa, oh-oh, oh-woah-whoa  \n[01:16.75] Kowareru hodo crush on you, kimi to nara fallin' in  \n[01:22.32] Aseru kimochi kotae wa mada, baby? Love is danger  \n[01:26.21] Shinchō ni naru hodo ni every piece of me, my pleasure  \n[01:30.46] Honey (honey) like roses (roses) mau yō ni sō  \n[01:33.96] Yureru omoi like a butterfly (butterfly), tokete ku moonlight  \n[01:38.76] Matteru nante mō iwasenai de  \n[01:42.82] Hey, you're my precious one, jibun no koe de step up, boom  \n[01:45.88] Tokei no hari ga futari made tick-tock  \n[01:50.03] Mahō no yō ni love in blooming (la-la-la-la)  \n[01:55.20] Kizukeba nukedasenai yo, I'm good to go  \n[01:59.74] (Go) yeah (yeah), tomedonaku muchū  \n[02:02.93] You're my crush, hajikeru bubble  \n[02:07.58] We can't stop, hikareau hodo (oh)  \n[02:12.10] Crush, takanaru kodō  \n[02:15.60] We can't stop, kimi to nara fallin' in love  \n[02:19.41] Whoa-whoa, oh-oh, woah-whoa, oh-oh  \n[02:23.33] Yume no naka odoritai sugar spot  \n[02:27.43] Whoa-whoa, oh-oh, oh-woah-whoa  \n[02:30.70] Kowareru hodo crush on you, kimi to nara fallin' in  \n[02:36.69] Oh, hey, DIVE, kore kara doko iku?  \n[02:40.76] I think I'm finally done with you  \n[02:44.57] Kowareru hodo love crush  \n[02:47.54] Tomedonaku muchū  \n[02:51.01] You're my crush, hajikeru bubble  \n[02:56.78] We can't stop, hikareau hodo (oh)  \n[03:01.31] Crush, takanaru kodō  \n[03:04.82] We can't stop, kimi to nara fallin' in love  \n[03:09.34] Crush on you, mō matenai blue moon (ah, yeah)  \n[03:13.25] I'm finally feeling like I got my freedom  \n[03:17.81] Crush on you, kimi mo muchū, yeah, yeah  \n[03:21.86] Love crush, you're my crush  \n[03:27.29]",
  "translatedLyrics": null
}
```
` ChatGPT API call took: 15544ms`

IN:
```
{
  "originalLyrics": "[00:29.07] 某年某月的某一天 就像一張破碎的臉\n[00:42.95] 難以開口道再見 就讓一切走遠\n[00:57.46] 這不是件容易的事 我們卻都沒有哭泣\n[01:11.80] 讓它淡淡的來 讓它好好的去\n[01:24.57] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[01:38.53] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[01:54.82] \n[02:22.90] 其實我沒有什麼好的本領能保護你\n[02:26.27] 對於愛的份量我卻總是疏於注意\n[02:29.95] 我該給自己來一個重擊\n[02:33.02] 還這十年一個痛快容易\n[02:36.20] 但你的笑容如此甜蜜這讓我怎麼能忘記\n[02:41.05] 淚在滴\n[02:43.41] 期盼著下場後的晚安\n[02:47.07] 卻留我獨自不寐輾轉\n[02:50.01] 如果說憑著思念我們就能回到過去\n[02:53.86] 我多想再給你一次擁抱和諾許\n[02:57.48] 我們不像你最愛的浪漫劇\n[03:00.69] 緊握與放手間掙扎猶豫\n[03:03.97] 但你的眼睛如此美麗這讓我怎麼能放棄\n[03:09.20] 去看清\n[03:11.13] 去觸摸你\n[03:16.41] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[03:29.61] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[03:43.78] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[03:57.40] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[04:11.05]",
  "languageCode": "zh",
  "romanize": false,
  "translate": true
}
```
OUT:
```
{
  "originalLyrics": "[00:29.07] 某年某月的某一天 就像一張破碎的臉\n[00:42.95] 難以開口道再見 就讓一切走遠\n[00:57.46] 這不是件容易的事 我們卻都沒有哭泣\n[01:11.80] 讓它淡淡的來 讓它好好的去\n[01:24.57] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[01:38.53] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[01:54.82] \n[02:22.90] 其實我沒有什麼好的本領能保護你\n[02:26.27] 對於愛的份量我卻總是疏於注意\n[02:29.95] 我該給自己來一個重擊\n[02:33.02] 還這十年一個痛快容易\n[02:36.20] 但你的笑容如此甜蜜這讓我怎麼能忘記\n[02:41.05] 淚在滴\n[02:43.41] 期盼著下場後的晚安\n[02:47.07] 卻留我獨自不寐輾轉\n[02:50.01] 如果說憑著思念我們就能回到過去\n[02:53.86] 我多想再給你一次擁抱和諾許\n[02:57.48] 我們不像你最愛的浪漫劇\n[03:00.69] 緊握與放手間掙扎猶豫\n[03:03.97] 但你的眼睛如此美麗這讓我怎麼能放棄\n[03:09.20] 去看清\n[03:11.13] 去觸摸你\n[03:16.41] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[03:29.61] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[03:43.78] 到如今年復一年 我不能停止懷念 懷念你懷念從前\n[03:57.40] 但願那海風再起 只為那浪花的手 恰似你的溫柔\n[04:11.05]",
  "languageCode": "ZH",
  "romanizedLyrics": null,
  "translatedLyrics": "[00:29.07] On a certain day in a certain month of a certain year, just like a broken face  \n[00:42.95] It's hard to say goodbye, just let everything drift away  \n[00:57.46] This is not an easy thing, yet we did not cry  \n[01:11.80] Let it come gently, let it go well  \n[01:24.57] As the years go by, I cannot stop reminiscing, missing you, missing the past  \n[01:38.53] I hope the sea breeze rises again, just for the hand of the waves, just like your tenderness  \n[01:54.82]  \n[02:22.90] Actually, I don't have any good skills to protect you  \n[02:26.27] I always neglect the weight of love  \n[02:29.95] I should give myself a hard hit  \n[02:33.02] To make these ten years a quick and easy release  \n[02:36.20] But your smile is so sweet, how can I forget it?  \n[02:41.05] Tears are falling  \n[02:43.41] Looking forward to the goodnight after the show  \n[02:47.07] Yet I am left alone, sleepless and tossing  \n[02:50.01] If we could return to the past through longing  \n[02:53.86] How I wish to give you one more hug and promise  \n[02:57.48] We are not like your favorite romantic drama  \n[03:00.69] Struggling and hesitating between holding on and letting go  \n[03:03.97] But your eyes are so beautiful, how can I give up?  \n[03:09.20] To see clearly  \n[03:11.13] To touch you  \n[03:16.41] As the years go by, I cannot stop reminiscing, missing you, missing the past  \n[03:29.61] I hope the sea breeze rises again, just for the hand of the waves, just like your tenderness  \n[03:43.78] As the years go by, I cannot stop reminiscing, missing you, missing the past  \n[03:57.40] I hope the sea breeze rises again, just for the hand of the waves, just like your tenderness  \n[04:11.05]"
}
```
⏱️ ChatGPT API call took: 13126ms

IN: (both trans + roman)
```
{
  "originalLyrics": "[00:08.77] Come on (ah-ah)\n[00:11.02] \n[00:25.83] Come on (let's go)\n[00:26.40] Stormy night, cloudy sky\n[00:29.83] In the moment, you and I\n[00:35.10] One more chance, 너와 나\n[00:38.62] 다시 한번 만나게 서로에게 향하게\n[00:43.73] My feeling's getting deeper\n[00:48.14] 내 심박수를 믿어\n[00:52.64] 우리 인연은 깊어\n[00:56.91] I gotta see the meaning of it (come on)\n[01:01.86] I don't know what we've done\n[01:04.09] 되돌아가긴 싫어, もう知っている\n[01:10.56] Don't know what we've been sold\n[01:12.90] 見つけられるよ, so it's sure (come on)\n[01:19.27] Golden moon, diamond stars\n[01:22.65] In a moment, you and I\n[01:28.08] Second chance, しょうがない\n[01:31.64] もう少し待って, 너와 내게 향하게 (ayy)\n[01:36.62] My feeling's getting deeper (deeper)\n[01:40.82] 내 심박수를 믿어 (믿어)\n[01:45.38] 우리 인연은 깊어 (you know)\n[01:49.77] I gotta see the meaning of it\n[01:54.74] I don't know what we've done\n[01:56.86] 되돌아가긴 싫어, もう知っている\n[02:03.65] Don't know what we've been sold\n[02:05.64] 見つけられるよ, so it's sure\n[02:12.14] \n[02:17.00] It's supernatural\n[02:20.82] \n[02:25.76] It's supernatural\n[02:29.21] 거짓말 안 할래, 너도 말해\n[02:34.17] Attention we should pay, to what is coming through\n[02:37.53] (To what is coming through) we had no idea (idea)\n[02:40.98] It's crystal clear, love is here, sitting next to you\n[02:48.67] \n[02:50.71] 私とあなた\n[02:56.14] Golden moon, diamond stars\n[02:59.82] In a moment, we unite\n[03:01.45] ",
  "languageCode": "ja",
  "romanize": true,
  "translate": true
}
```

OUT: 
```
{
  "originalLyrics": "[00:08.77] Come on (ah-ah)\n[00:11.02] \n[00:25.83] Come on (let's go)\n[00:26.40] Stormy night, cloudy sky\n[00:29.83] In the moment, you and I\n[00:35.10] One more chance, 너와 나\n[00:38.62] 다시 한번 만나게 서로에게 향하게\n[00:43.73] My feeling's getting deeper\n[00:48.14] 내 심박수를 믿어\n[00:52.64] 우리 인연은 깊어\n[00:56.91] I gotta see the meaning of it (come on)\n[01:01.86] I don't know what we've done\n[01:04.09] 되돌아가긴 싫어, もう知っている\n[01:10.56] Don't know what we've been sold\n[01:12.90] 見つけられるよ, so it's sure (come on)\n[01:19.27] Golden moon, diamond stars\n[01:22.65] In a moment, you and I\n[01:28.08] Second chance, しょうがない\n[01:31.64] もう少し待って, 너와 내게 향하게 (ayy)\n[01:36.62] My feeling's getting deeper (deeper)\n[01:40.82] 내 심박수를 믿어 (믿어)\n[01:45.38] 우리 인연은 깊어 (you know)\n[01:49.77] I gotta see the meaning of it\n[01:54.74] I don't know what we've done\n[01:56.86] 되돌아가긴 싫어, もう知っている\n[02:03.65] Don't know what we've been sold\n[02:05.64] 見つけられるよ, so it's sure\n[02:12.14] \n[02:17.00] It's supernatural\n[02:20.82] \n[02:25.76] It's supernatural\n[02:29.21] 거짓말 안 할래, 너도 말해\n[02:34.17] Attention we should pay, to what is coming through\n[02:37.53] (To what is coming through) we had no idea (idea)\n[02:40.98] It's crystal clear, love is here, sitting next to you\n[02:48.67] \n[02:50.71] 私とあなた\n[02:56.14] Golden moon, diamond stars\n[02:59.82] In a moment, we unite\n[03:01.45] ",
  "languageCode": "JA",
  "romanizedLyrics": "[00:08.77] Come on (ah-ah)  \n[00:11.02]  \n[00:25.83] Come on (let's go)  \n[00:26.40] Stormy night, cloudy sky  \n[00:29.83] In the moment, you and I  \n[00:35.10] One more chance, neo wa na  \n[00:38.62] dasi hanbeon mannage seoroege hyanghage  \n[00:43.73] My feeling's getting deeper  \n[00:48.14] nae simbaksureul mideo  \n[00:52.64] uri inyeon-eun gipeo  \n[00:56.91] I gotta see the meaning of it (come on)  \n[01:01.86] I don't know what we've done  \n[01:04.09] doedol-agagin silheo, mou shitteiru  \n[01:10.56] Don't know what we've been sold  \n[01:12.90] mitsukerareru yo, so it's sure (come on)  \n[01:19.27] Golden moon, diamond stars  \n[01:22.65] In a moment, you and I  \n[01:28.08] Second chance, shouganai  \n[01:31.64] mou sukoshi matte, neo wa naege hyanghage (ayy)  \n[01:36.62] My feeling's getting deeper (deeper)  \n[01:40.82] nae simbaksureul mideo (mideo)  \n[01:45.38] uri inyeon-eun gipeo (you know)  \n[01:49.77] I gotta see the meaning of it  \n[01:54.74] I don't know what we've done  \n[01:56.86] doedol-agagin silheo, mou shitteiru  \n[02:03.65] Don't know what we've been sold  \n[02:05.64] mitsukerareru yo, so it's sure  \n[02:12.14]  \n[02:17.00] It's supernatural  \n[02:20.82]  \n[02:25.76] It's supernatural  \n[02:29.21] geojitmal an halrae, neodo malhae  \n[02:34.17] Attention we should pay, to what is coming through  \n[02:37.53] (To what is coming through) we had no idea (idea)  \n[02:40.98] It's crystal clear, love is here, sitting next to you  \n[02:48.67]  \n[02:50.71] watashi to anata  \n[02:56.14] Golden moon, diamond stars  \n[02:59.82] In a moment, we unite  \n[03:01.45]",
  "translatedLyrics": "Come on (ah-ah)  \nCome on (let's go)  \nStormy night, cloudy sky  \nIn the moment, you and I  \nOne more chance, you and I  \nLet's meet once more, heading towards each other  \nMy feeling's getting deeper  \nTrust my heartbeat  \nOur connection is deep  \nI gotta see the meaning of it (come on)  \nI don't know what we've done  \nI don't want to go back, I already know  \nDon't know what we've been sold  \nI can find it, so it's sure (come on)  \nGolden moon, diamond stars  \nIn a moment, you and I  \nSecond chance, it can't be helped  \nJust wait a little longer, heading towards you and me (ayy)  \nMy feeling's getting deeper (deeper)  \nTrust my heartbeat (trust)  \nOur connection is deep (you know)  \nI gotta see the meaning of it  \nI don't know what we've done  \nI don't want to go back, I already know  \nDon't know what we've been sold  \nI can find it, so it's sure  \nIt's supernatural  \nIt's supernatural  \nI won't lie, you say it too  \nAttention we should pay, to what is coming through  \n(To what is coming through) we had no idea (idea)  \nIt's crystal clear, love is here, sitting next to you  \nWith you  \nGolden moon, diamond stars  \nIn a moment, we unite"
}
```
⏱️ ChatGPT API call took: 19945ms

## 💬 Additional Comments

Currently a bit slow, optimization will have to be looked into (another issue).
There is dotnet openai library which is like a wrapper, but I found it harder to use? Using HTTP client raw json turned out to be easier. So far 9 requests costs < 0.01$ so pretty sustainable.

### CODE REVIEW

File: ChordKTV/Services/Service/ChatGptService.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            messages = new object[]
+            {
+                    new { role = "system", content = "You are a helpful assistant that translates LRC formatted lyrics into english and/or romanized version." },
+                    new { role = "user", content = prompt }
+            },
+            temperature = 0
+        };
+
+        var jsonRequest = JsonSerializer.Serialize(requestBody);
+        Console.WriteLine("🔍 JSON Request: " + jsonRequest);
```
Comment (parthivshaji) [inline ChordKTV/Services/Service/ChatGptService.cs:63]: For now this logging is fine, but thoughts on using Microsoft.Extensions.Logging instead? Would probably save us a lot of pain in the future in the deployment stage if we start using it now

Comment (darint916) [reply ChordKTV/Services/Service/ChatGptService.cs]: Fixed, swapped to logging

**Review by parthivshaji — APPROVED** [review]
Good work, (afaik) we need to configure the logger to either log to console or to a file in Program.cs, but we can do that later. Approving for now.

**Comment by darint916** [comment]
#21 added for logging TODO
